ultralytics/data/augment.py

```diff
@@ -129,10 +129,10 @@ class Mosaic(BaseMixTransform):
         n (int, optional): The grid size, either 4 (for 2x2) or 9 (for 3x3).
     """
 
-    def __init__(self, dataset, imgsz=640, p=1.0, n=4):
+    def __init__(self, dataset, imgsz=640, p=1.0, n=3):
```

Comment:
Don't change default value, your contribution has to be an addition, not a default setting change

Reply:
I just done this https://github.com/Leo-Moooon/ultralytics/pull/1.